content/influxdb/clustered/install/configure-cluster/use-helm.md

```diff
@@ -0,0 +1,964 @@
+  - Token endpoint
+
+InfluxDB is deployed to a Kubernetes namespace which, throughout the following
+installation procedure, is referred to as the _target_ namespace.
+For simplicity, we assume this namespace is `influxdb`, however
+you may use any name you like.
+
+### AppInstance resource
+
+The InfluxDB installation, update, and upgrade processes are driven by editing
+and applying a [Kubernetes custom resource (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+called `AppInstance`.
+The `AppInstance` CRD is included in the InfluxDB Clustered Helm chart and can
+be configured by applying custom settings in the `values.yaml` included in the
+chart.
+
+The `AppInstance` resource contains key information, such as:
+
+- Name of the target namespace
+- Version of the InfluxDB package
+- Reference to the InfluxDB container registry pull secrets
+- Hostname where the InfluxDB API is exposed
+- Parameters to connect to [external prerequisites](/influxdb/clustered/install/prerequisites/)
+
+### Kubit operator
+
+The InfluxDB Clustered Helm chart also includes the
+[`kubecfg kubit` operator](https://github.com/kubecfg/kubit) (maintained by InfluxData)
+which simplifies the installation and management of the InfluxDB Clustered package.
+It manages the application of the jsonnet templates used to install, manage, and
+update an InfluxDB cluster.
+
+## Configure your cluster
+
+1.  [Install Helm](#install-helm)
+2.  [Create a values.yaml file](#create-a-valuesyaml-file)
+3.  [Create a namespace for InfluxDB](#create-a-namespace-for-influxdb)
+4.  [Configure access to the InfluxDB container registry](#configure-access-to-the-influxdb-container-registry)
+5.  [Set up cluster ingress](#set-up-cluster-ingress)
+6.  [Modify the configuration file to point to prerequisites](#modify-the-configuration-file-to-point-to-prerequisites)
+7.  [Provide a custom certificate authority bundle](#provide-a-custom-certificate-authority-bundle)
+    <em class="op65">(Optional)</em>
+
+### Install Helm
+
+If you haven't already, [install Helm](https://helm.sh/docs/intro/install/) on
+your local machine.
+
+### Create a values.yaml file
+
+Download or copy the base `values.yaml` for the InfluxDB Clustered Helm chart
+from GitHub and store it locally. For example--if using cURL:
+
+```bash
+curl -O https://raw.githubusercontent.com/influxdata/helm-charts/master/charts/influxdb3-clustered/values.yaml
+```
+
+Or you can copy the default `values.yaml` from GitHub:
+
+<a href="https://github.com/influxdata/helm-charts/blob/master/charts/influxdb3-clustered/values.yaml" class="btn github">View values.yaml on GitHub</a>
+
+### Create a namespace for InfluxDB
+
+Create a namespace for InfluxDB. For example, using `kubectl`:
+
+```sh
+kubectl create namespace influxdb
+```
+
+If you use a namespace name other than `influxdb`, update the `namespaceOverride`
+field in your `values.yaml` to use your custom namespace name.
```

Comment:
Is this necessary with Helm or does Helm create the namespace for you when you deploy the chart?

Reply:
When using `helm install` or helm upgrade --install`, the `--create-namespace` flag will create the specified namespace.

We can use the `--create-namespace` flag instead of `kubectl create namespace influxdb`, but we'll still want to keep this section advising users they should set a custom namespace in the `values.yaml` file.

Reply:
Ok, some of the instructions after this (like adding secrets) require an existing namespace, so I assume this step is still required at this point.